### PR TITLE
[FSDE-12538] Migrate price component to vue documentation

### PR DESCRIPTION
**Pull request by soujanreddy**
@jllr This PR requires story number to tag price component.

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://a74955fe9386aab007779a8d786309f3997deaa9-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-280-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/8/. :x:

**Review by evansito — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/9/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f62830eca515f310eaf1e27feaceadd041b3d89a-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-280-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/10/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://4b56cebec0f8c76d18b46dbdd7262c2774e75ea0-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-280-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/12/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/13/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/14/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://596882a0c446d329ed75a58d35fa7cb09c00c9cd-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-280-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/15/. :white_check_mark:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/17/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-280/18/. :x:

File: src/documentation/pages/components/price/examples/_base.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" padding="-p--0" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/price/examples/_base.vue:2]:
```suggestion
  <ComponentExample title="Base" id="base" :tabs="exampleTabs">
```

File: src/documentation/pages/components/price/examples/_base.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    .-p--3(slot="example")
+      .chi-price.-m--3
```
Comment (jllr) [inline src/documentation/pages/components/price/examples/_base.vue:4]:
```suggestion
    .chi-price.-m--3(slot="example")
```

File: src/documentation/pages/components/price/_accessibility.vue [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<template lang="pug">
+  div
```
Comment (jllr) [inline src/documentation/pages/components/price/_accessibility.vue:2]: Please update with the latest Accessibility component that has done Evan

File: src/documentation/pages/components/price/_properties.vue [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<template lang="pug">
+  div
```
Comment (jllr) [inline src/documentation/pages/components/price/_properties.vue:2]: Please update with the latest Properties component that has done Evan

**Review by jllr — CHANGES_REQUESTED** [review]

File: src/documentation/pages/components/price/examples/_base.vue (hunk 1 above)
Comment (soujanreddy) [reply src/documentation/pages/components/price/examples/_base.vue:2]: I think we can update it to -p--3 to match with the current padding in production.

File: src/documentation/pages/components/price/examples/_base.vue [hunk 5]
```diff
@@ -0,0 +1,48 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" padding="-p--3" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/price/examples/_base.vue:2]:
```suggestion
  <ComponentExample title="Base" id="base" :tabs="exampleTabs">
```

File: src/documentation/pages/components/price/examples/_base.vue (hunk 1 above)
Comment (jllr) [reply src/documentation/pages/components/price/examples/_base.vue]: -p--3 is the padding marked by default in the new doc system, so it's not neccessary to include it

**Review by jllr — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Pull request merged**